bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/util/SemesterData.scala

```diff
@@ -111,7 +111,7 @@ object SemesterData {
 
   def current(site: Site, currentTime: Long = System.currentTimeMillis()): SemesterData = semesters(site, currentTime, currentTime).head
 
-  def next(site: Site): SemesterData = semesters(site, System.currentTimeMillis(), System.currentTimeMillis() + TimeUtils.days(185))(1)
+  def next(site: Site): SemesterData = semesters(site, System.currentTimeMillis(), System.currentTimeMillis() + 185.days.toMillis)(1)
```

Comment:
185 days? Maybe we should have a constant for that

Reply:
Is this correct? What about leap years?

Reply:
I'm not even sure if the `semesters` method is doing what it claims to be doing:
```scala
  /**
   * Gets all available semesters covering the given time frame.
   * @return
   */
  def semesters(site: Site, start: Long, end: Long): Seq[SemesterData] =
    semesters(site).
      filter(sd => sd.end > start && sd.start < end)
```

It seems like this actually gets all the semesters intersecting the given time frame, rather than **covering** the given time frame?

I would think a semester would cover `[start, end]` iff `sd.start <= start && sd.end >= end`?

I'm kind of confused by this original logic.

Reply:
I think 185 would always work, no? Even for leap years, 366 / 2 = 183, so 185 should be sufficient to advance a semester. My concern would be the potential to advance two semesters accidentally if we were on the last day of a semester.

This code definitely doesn't seem appropriate, but I think trying to look at its actual functionality and correct it if it is wrong should be a separate JIRA task.